uiowa/uiowa
# Feature: intranet search

Resolves https://github.com/uiowa/uiowa/issues/4244. 

This pr:

- Turns on Search API for sitenow_intranet
- Adds a custom search view for sitenow_intranet
- Adds a search region to `uids_base` and a corresponding search block for `uiowa_search`
- Makes the custom search form use placeholder text from the "Search label" set in the form

# How to test
```
blt frontend
blt drupal:install --site default
drush en sitenow_intranet
drush cim
```
- Test that the search block takes you to `/search` which is a custom view search page. 
- Run cron and verify that the site search works by searching for `ITS`.
